images/base-ide/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Dependency installation stage to maximise caching
+FROM node:20-bullseye AS install-deps
```

Comment:
For our [current Dockerfile](https://github.com/eclipse-theia/theia-ide/blob/master/browser.Dockerfile), we are using node 22. This should be used for this Dockerfile and all other examples as well.

To make it more transparent what has changed could you:
- Make a commit that moves the `browser.Dockerfile` to this location and make your adaptions.
- Create a second commit that readds a `browser.Dockerfile` that produces the same application as before but uses the base-ide image? (I think there it would be fine to just copy over the package.json)